addons/mail/static/src/components/composer/composer.scss

```diff
@@ -231,8 +245,12 @@
 
 .o_Composer_coreMain:not(.o-composer-is-compact) {
     background: white;
-    border-radius: 5px;
     border: 1px solid lighten(gray('400'), 5%);
+
+    // textarea should not be rounded when there is subject field above
+    &:not(.o-composer-is-extended) {
+        border-radius: 5px;
```

Comment:
Possible to use one of the $border-radius ?

Reply:
Don't know, didn't change the value, only restricted it, I can check

Reply:
It does not change a lot but even `$border-radius-lg` is not enough to be exactly the same as `5px` so I don't know if we reduce the radius or not ?

Reply:
If it looks good with new value you can change it. If you are not sure feel free to post screenshot

Reply:
`$border-radius-lg`
![image](https://user-images.githubusercontent.com/48204996/85406527-3cd16600-b562-11ea-9a3d-1bdd65d614f7.png)

`5px`
![image](https://user-images.githubusercontent.com/48204996/85406591-507ccc80-b562-11ea-9cb5-2610d73f04a9.png)